wp-cache-export.php

```diff
@@ -0,0 +1,472 @@
+<?php
+
+/**
+* Export & Import WP Super Cache settings.
+*
+* This class handles the exporting and importing of WP Super Cache settings. Exports are downloaded
+* as a JSON file with properties that mirror the WP Super Cache configuration file. Imports are handled
+* by uploading an exported JSON file and overwriting the WP Super Cache configuration file with the
+* imported parameters. A backup of the original configuration file is kept.
+*
+* @package  WP Super Cache
+* @since 1.4.4
+*/
+
+class WP_Super_Cache_Export {
+
+  const TITLE = 'WP_Super_Cache_Export.json';
+
+  const NAME = '_wp_super_cache_export';
+
+  const IMPORT_NONCE = 'wp-super-cache-import';
+
+  const EXPORT_NONCE = 'wp-super-cache-export';
+
+  const RESTORE_NONCE = 'wp-super-cache-restore';
+
+  const REMOVE_NONCE = 'wp-super-cache-remove';
+
+  const OPTIONS = array(
+    'ossdl_cname',
+    'ossdl_https',
+    'ossdl_off_cdn_url',
+    'ossdl_off_exclude',
+    'ossdl_off_include_dirs',
+    'preload_cache_counter',
+    'supercache_last_cached',
+    'supercache_stats',
+    'wpsupercache_count',
+    'wpsupercache_gc_time',
+    'wpsupercache_start',
+  );
+
+  static $cache_config_file;
+
+  static $cache_config_file_backup;
+
+  static $MESSAGES = array();
+
+
+  /**
+   * Instantiate the class to enable the importing and exporting features for the WP Super Cache plugin.
+   *
+   * Caches the location of the WP Super Cache plugin configuration file and the sample configuration file.
+   * Adds two actions to the WP Super Cache admin page load that determine if an import or export
+   * should be preformed.
+   *
+   * Defines the static $MESSAGES array that displays the success or error messages in the upload form.
+   *
+   * @uses  $wp_cache_config_file The location of the configuration file.
+   * @uses  $wp_cache_config_file_sample The location of the sample configuration file.
+   *
+   * @since  1.4.4
+   */
+
+  function __construct() {
+    global $wp_cache_config_file, $wp_cache_config_file_sample;
+
+    self::$cache_config_file = $wp_cache_config_file;
+    self::$cache_config_file_backup = str_replace( '.php', '-backup.php', self::$cache_config_file );
+
+    self::$MESSAGES = array(
+      0 =>  array( 'success', __( 'Settings imported', 'wp-super-cache' ) ),
+      1 =>  array( 'warning', __( 'Please upload an exported WP Super Cache settings file.', 'wp-super-cache' ) ),
+      2 =>  array( 'error', __( 'Unable to import the uploaded JSON file. Please export the settings and import them again.', 'wp-super-cache' ) ),
```

Comment:
I actually felt the same way about that particular message. I made the message more specific 01489641.